### PR TITLE
Initial PMetric library

### PR DESCRIPTION
**Pull request by bschaeffer**

### CODE REVIEW

File: pmetric.gemspec [hunk 1]
```diff
@@ -21,9 +21,11 @@ Gem::Specification.new do |spec|
 
   # Monitoring Gems
   spec.add_dependency 'influxdb', '~> 0.3.14'
-  spec.add_dependency 'activesupport', '>= 3.2.22', '< 5.0.0'
+  spec.add_dependency 'activesupport', '>= 3.2.22', '< 6'
 
-  spec.add_development_dependency "bundler", "~> 1.13"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency 'appraisal'
```
Comment (ridiculous) [inline pmetric.gemspec:26]: this is nice

File: spec/lib/pmetric_spec.rb [hunk 2]
```diff
@@ -0,0 +1,116 @@
+require 'spec_helper'
```
Comment (ridiculous) [inline spec/lib/pmetric_spec.rb:1]: where is the spec_helper?

**Review by ridiculous — APPROVED** [review]
Looks good. Can we also look at further limiting the files included in the bundled gem? We don't need the README, bin/console, etc

File: spec/lib/pmetric_spec.rb (hunk 2 above)
Comment (ridiculous) [reply spec/lib/pmetric_spec.rb:1]: Nevermind, I see it's already commited

**Comment by bschaeffer** [comment]
@ridiculous just saw your comments... I will configure the gemspec to include only the `lib` files.

**Pull request merged**